dcos/tools.go

```diff
@@ -7,13 +7,31 @@ import (
 	"io/ioutil"
 	"net/http"
 	"os"
+	"strconv"
 	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
 	"github.com/sirupsen/logrus"
 
 	"github.com/dcos/dcos-diagnostics/util"
 )
 
+var (
+	findMasterNodesHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
```

Comment:
Would it make sense to put this and `doRequestHistogram` into the `Tools` struct so we have less global variables flying around?!